ShopLens/ShopLensForms/ShopLens.cs

```diff
@@ -127,7 +132,33 @@ private void PAUSE_Click(object sender, EventArgs e)
 
         private void CAPTURE_Click(object sender, EventArgs e)
         {
+            if (live_video.Image != null)
```

Comment:
Wait a second 🕐 ... Is this method just copy-paste 🤔 🤔 🤔 

Remember, we want to be DRY 🏜 🐪 (Don't Repeat Yourself - OOP principle) 

But, in all seriousness, this method is just here for testing purposes and will be removed? Or will we still have it for testing purposes?

Either way, I don't think we should merge two copy-pasted methods in master, not because it's very bad, but because code hygiene. So we should either throw it out, or make it DRY.

Reply:
Yeah, you're right, I'll make it so that the what is this command event method runs this method. That will probably make it 🌵 🐫 🏜  DRY  🏜 🐫 🌵 enough.